dot/network/service.go

```diff
@@ -581,7 +581,7 @@ func (s *Service) readStream(stream libp2pnetwork.Stream, decoder messageDecoder
 	for {
 		tot, err := readStream(stream, msgBytes[:])
 		if err == io.EOF {
```

Comment:
would be nice to use `errors.Is(err, io.EOF)`?

Reply:
Done